moveit_core/transforms/src/transforms.cpp

```diff
@@ -37,14 +37,14 @@
 #include <moveit/transforms/transforms.h>
 #include <tf2_eigen/tf2_eigen.h>
 #include <boost/algorithm/string/trim.hpp>
-#include "rclcpp/rclcpp.hpp"
+#include <rclcpp/logging.hpp>
 
 namespace moveit
 {
 namespace core
 {
 // Logger
-static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_transforms.transforms");
+static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit.core.transforms");
```

Comment:
Why is this moveit.core.transforms and not moveit.core.transforms.transforms?

If it is because the library and source file name are redundant and there are no other source files in the library, that work for me. Otherwise, if there are other source files, my preference would be to keep the redundant file name. Dropping it would make filtering in/out transforms.cpp logger statements less straightforward (but not impossible).

Reply:
Yes, we are removing redundancy in the logger names. Your point about filtering other files is definitely valid, though. Is this the case with the transforms package?

Reply:
transforms is the only source file in the transforms library, but I wanted to see if there was an decided policy before poking around further for logger names in the other case. Personally, I'd rather keep a longer, redundant logger name (2x .transforms) than have the overhead of remembering edge cases in a redundancy name rule.

Reply:
Well, the policy was to reduce redundancy in the file names if possible (i.e. if they are unambiguous). I don't think that we would have to remember any logger names, since they are mostly used as unique pointers to the source files or semantic groups. My only concern is that added source files might require changing logger names of already existing ones.